tsconfig.json

```diff
@@ -0,0 +1,30 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+ 
+{
+    "compilerOptions": {
+        "baseUrl": "./src/client/app/",
+        "outDir": "./src/client/public/",
+        "sourceMap": true,
+        "module": "commonjs",
+        "target": "es5",
+		"jsx": "react",
+		"allowJs": true,
```

Comment:
Because we have some JavaScript files in `common/` that need to be included. They are typed, and if untyped JS is included the implicit `any` checking will catch it.